src/coreclr/jit/lower.cpp

```diff
@@ -5588,6 +5588,25 @@ void Lowering::InsertPInvokeCallEpilog(GenTreeCall* call)
 {
     JITDUMP("======= Inserting PInvoke call epilog\n");
 
+    GenTree* insertionPoint = call;
+
+#ifdef SWIFT_SUPPORT
+    // For Swift calls that require error handling, ensure the GT_SWIFT_ERROR node
+    // that consumes the error register is the call node's successor.
+    // This is to simplify logic for marking the error register as busy in LSRA.
+    if ((call->gtCallMoreFlags & GTF_CALL_M_SWIFT_ERROR_HANDLING) != 0)
+    {
+        do
+        {
+            insertionPoint = insertionPoint->gtNext;
+            assert(insertionPoint != nullptr);
+        } while (!insertionPoint->OperIs(GT_SWIFT_ERROR));
+
+        BlockRange().Remove(insertionPoint);
+        BlockRange().InsertAfter(call, LIR::SeqTree(comp, insertionPoint));
```

Comment:
```suggestion
        BlockRange().InsertAfter(call, insertionPoint);
```

Sequencing is only necessary when you create a tree with multiple nodes in it that need to be linearized.